src/types.tsx

```diff
@@ -242,21 +242,21 @@ export interface FormikRegistration {
 export type FormikContext<Values> = FormikProps<Values> &
   Pick<FormikConfig<Values>, 'validate' | 'validationSchema'>;
 
-export interface SharedRenderProps<T> {
+export interface SharedRenderProps<P> {
   /**
```

Comment:
I would not alter type name. It is a common among TS devs (and others) to reference to generic by `T` type. Less changes, more likely PR gets merged ;)

Reply:
True. I'll change it.

Reply:
On second thought… type types which are passed to `SharedRenderProps` are props which are used on `React.ComponentType`. This was the reason that I used "P" instead of "T". All the examples I see where types are passed to `React.ComponentType` they are passed using "P" or some longer variable with the word "props" in it somewhere.

Reply:
Here's the actual `ComponentType` type from DefinitelyTyped:

```ts
type ComponentType<P = {}> = ComponentClass<P> | FunctionComponent<P>;
```
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L63

Note that it uses `P` not `T`